src/test/java/io/vertx/openapi/test/E2ETest.java

```diff
@@ -86,6 +88,38 @@ void testExtractPath(String basePath, VertxTestContext testContext) {
     }).onFailure(testContext::failNow);
   }
 
+  //These types come from: https://spec.openapis.org/oas/latest.html#special-considerations-for-multipart-content
+  // 1 primitive "petId"
+  // 1 complex "json"
+  // 1 primitive with content encoding aka png base64 encoded.
+  @Test
+  @Timeout(value = 2, timeUnit = TimeUnit.SECONDS)
+  @DisplayName("Send a multipart/form-data request")
+  public void sendMultipartFormDataRequest(VertxTestContext testContext) {
+    Path path = getRelatedTestResourcePath(E2ETest.class).resolve("multipart.txt");
+
+
+    setupContract("", testContext).compose(v -> createValidationHandler(req -> {
+      testContext.verify(() -> {
+        assertThat(req.getBody()).isNotNull();
+        JsonObject jsonReq = req.getBody().getJsonObject();
+        assertThat(jsonReq.getLong("petId")).isEqualTo(1234L);
+        assertThat(jsonReq.getJsonObject("petMetadata").toString())
+          .isEqualTo("{\"friends\":[123,456,789],\"contactInformation\":{\"name\":\"Example\",\"email\":\"example@example.com\",\"phone\":\"5555555555\"}}");
+        assertThat(jsonReq.getBuffer("petPicture")).isNotNull();
+        testContext.completeNow();
+      });
+      return ValidatableResponse.create(201);
+    },contract.operation("uploadPet").getOperationId(), testContext))
+      .compose(v -> createRequest(HttpMethod.POST, "/pets/upload"))
+      .map(request -> request.putHeader(HttpHeaders.CONTENT_TYPE, "multipart/form-data; boundary=4ad8accc990e99c2"))
+      .map(request -> request.putHeader(HttpHeaders.CONTENT_DISPOSITION, ""))
+      .compose(request -> request.send(vertx.fileSystem().readFileBlocking(path.toString())))
+      .onFailure(testContext::failNow);
+  }
+
+
+
```

Comment:
Are these three empty lines intended?

Reply:
Bad habit of adding extra new lines between tests. I removed the excess ones